### DISCUSSION TITLE
добавить управление перезарядкой оружия

### DISCUSSION THREAD

**Discussion by zjusttoxic** [Sokol-Games-GmbH/PixelCombats.PublicApi#27] [Ideas]
**1. Перезарядка оружия.**
С помощью какого-нибудь метода с названием Reload() оружие будет перезаряжаться, а именно восполняться боезапас. 
Пример: Inventory.GetContext(player).Main.Reload();

позже может дополню

**Comment by kkohno** [comment]
идея хорошая
